operator/README.md

```diff
@@ -0,0 +1,175 @@
+```
+
+### Deployment CRD API
+
+Current PMEM-CSI Deployment object supports below API:
+
+#### Deployment
+
+|Field | type | Description |
+|---|---|---|
+| apiVersion | string  | `pmem-csi.intel.com/v1alpha1`|
+| kind | string | `Deployment`|
+| metadata | [ObjectMeta](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status) | Standard objects metadata |
+| spec | [DeploymentSpec](#deployment-spec) | Specification of the desired behavior of the deployment |
+
+#### DeploymentSpec
+
+|Field | type | Description | Default Value |
+|---|---|---|---|
+| driverName | string | Unique CSI driver name to use | `pmem-csi.intel.com` |
+| image | string | PMEM-CSI docker image name used for the deployment | the same version as the operator |
+| provisionerImage | string | [CSI provisioner](https://kubernetes-csi.github.io/docs/external-provisioner.html) docker image name | latest [external provisioner](https://kubernetes-csi.github.io/docs/external-provisioner.html) stable release image<sup>1</sup> |
+| registrarImage | string | [CSI node driver registrar](https://github.com/kubernetes-csi/node-driver-registrar) docker image name | latest [node driver registrar](https://kubernetes-csi.github.io/docs/node-driver-registrar.html) stable release image<sup>1</sup> |
+| pullPolicy | string | Docker image pull policy. either one of `Always`, `Never`, `IfNotPresent` | `IfNotPresent` |
+| logLevel | integer | PMEM-CSI driver logging level | 3 |
+| deviceMode | string | Device management mode to use. Supports one of `lvm` or `direct` | `lvm`
+| controllerResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for controller pod |
+| nodeResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for the pods running on node(s) |
+| registryCert | string | Encoded tls certificate signed by a certificate authority used for driver's controller registry server | generated by operator self-signed CA |
+| nodeControllerCert | string | Encoded tls certificate signed by a certificate authority used for driver's node controllers | generated by operator self-signed CA |
+| registryKey | string | Encoded RSA private key used for signing by `registryCert` | generated by the operator |
+| nodeControllerKey | string | Encoded RSA private key used for signing by `nodeControllerCert` | generated by the operator |
+| caCert | string | Certificate of the CA by which the `registryCert` and `controllerCert` are signed | self-signed certificate generated by the operator |
+| nodeSelector | string map | [Labels to use for selecting Nodes](../README.md#run-pmem-csi-on-kubernetes) on which PMEM-CSI driver should run. | `{ "storage": "pmem" }`|
+
+<sup>1</sup> Image versions are depend on the Kubernetes cluster version.
```

Comment:
s/are depend/depend/

It's worth explaining here that the operator figures that our itself for the default values whereas users have to handle this themselves when overriding the values.

Reply:
Added the suggested explanation about default versions.